client/lib/login/index.js

```diff
@@ -173,12 +164,7 @@ export function getSignupUrl( currentQuery, currentRoute, oauth2Client, locale,
 		return `${ signupUrl }/${ signupFlow }`;
 	}
 
-	if (
-		isFromMigrationPlugin ||
-		isFromPublicAPIConnectFlow ||
-		( includes( redirectTo, 'action=jetpack-sso' ) && includes( redirectTo, 'sso_nonce=' ) ) ||
-		redirectTo
-	) {
```

Comment:
Most of this condition is redundant since https://github.com/Automattic/wp-calypso/pull/83026/files#r1361480491 because if `redirectTo` is non-empty, the entire condition is `true`.